doc/nrf/working_with_nrf/nrf70/developing/regulatory_support.rst

```diff
@@ -0,0 +1,328 @@
+   * - 140
+     - 5690
+     - 5710
+     - 5700
+     - U-NII-2C
+   * - 144
+     - 5710
+     - 5730
+     - 5720
+     - U-NII-2C
+   * - 149
+     - 5735
+     - 5755
+     - 5745
+     - U-NII-3
+   * - 153
+     - 5755
+     - 5775
+     - 5765
+     - U-NII-3
+   * - 157
+     - 5775
+     - 5795
+     - 5785
+     - U-NII-3
+   * - 161
+     - 5795
+     - 5815
+     - 5805
+     - U-NII-3
+   * - 165
+     - 5815
+     - 5835
+     - 5825
+     - U-NII-3
+   * - 169
+     - 5835
+     - 5855
+     - 5845
+     - U-NII-4
+   * - 173
+     - 5855
+     - 5875
+     - 5865
+     - U-NII-4
+   * - 177
+     - 5875
+     - 5895
+     - 5885
+     - U-NII-4
+
+Conformance to regulatory restrictions
+**************************************
+
+Once a regulatory domain is configured, the following restrictions are imposed on the operation of the nRF70 Series device:
+
+1. Channels: The device will operate only on the channels allowed in that country.
+#. Scan mode: The device will use Active scans where allowed and resort to Passive scans on channels where Active scans are not allowed.
+#. DFS: DFS operation is enabled on the channel.
+   The device will operate in the following two states:
+
+   * Scan state: The channel operation must be in the Passive mode.
+   * Connected state: The device will switch to another channel as directed by the Access Point (AP) in the event of radar detection.
+#. Transmit power: The maximum transmit power on each channel is restricted to a value allowed for that channel in that country.
+
+Without user configuration, the nRF70 Series device will default to operating with world regulatory rules and utilize regulatory hints extracted from the AP beacon packet in order to adjust to more appropriate regional rules.
+
+
+Configuration options
+*********************
+
+You can configure the regulatory domain through build time or run time.
+
+Build time
+==========
+
+Use the :kconfig:option:`CONFIG_NRF700X_REG_DOMAIN` Kconfig option to set the regulatory region.
+The regulatory region will take a ISO/IEC alpha-2 country code for the country in which the device is expected to operate.
+
+Run time
+========
+
+You can also set the regulatory domain using either an API or a shell command as shown in the following table:
+
+.. list-table:: Wi-Fi regulatory domain network management APIs
+   :header-rows: 1
+
+   * - Network management APIs
+     - Command
+     - Description
+     - Expected output
+   * - | net_mgmt(NET_REQUEST_WIFI_CMD_REG_DOMAIN, <XY>, force=true)
+       | (Set option 1)
+     - ``wifi reg_domain <XY> -f``
+     - Override country code regulatory hints
+     - Wi-Fi regulatory domain is set to: <XY>
+   * - | net_mgmt(NET_REQUEST_WIFI_CMD_REG_DOMAIN, <XY>)
+       | (Set option 2)
+     - ``wifi reg_domain <XY>``
+     - Set regulatory domain using regulatory hints from the AP beacon, else set regulatory domain to XY
+     - Wi-Fi regulatory domain is set based on AP beacon hints or to XY
```

Comment:
Keep the less than / greater than brackets around XY. Missing in comment since did not know how to make visible....

Reply:
Remove the lines in the point 3: The device will be operated in two states as below.
Scan state: The channel operation should be in Passive mode.

and modify the following line:
In the Connected state, the device will switch to another channel as directed by the Access Point (AP) in the event of radar detection.